### PR TITLE
Adding directions on updating rust toolchain.

### PR DESCRIPTION
**Pull request by efrisch**
## Description

Fixes pyrsia/pyrsia#592

This PR documents how to update the rust toolchain.

## Screenshots (optional)
Screen Shot 2022-04-15 at 7 56 58 AM

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).
- [x] I've requested a review  from `pyrsia/collaborators`.

### Code Contributions

- [x] I've built the code `cargo build --all-targets` successfully.
- [x] I've run the unit tests `cargo test --workspace` and everything passes.

### CODE REVIEW

File: docs/dev_workflow.md [hunk 1]
```diff
@@ -64,3 +63,11 @@ cargo audit
 
 It's strongly encouraged to run a full test to make sure interaction with external tools are not broken.
 You can follow the [Local Development Setup](local_dev_setup.md) guide to perform this.
+
+## How-to Update
+
+We currently target the most current stable version of the rust toolchain. Perioudically execute
```
Comment (prince-chrismc) [inline docs/dev_workflow.md:69]:
```suggestion
We currently target the most current stable version of the rust toolchain. Periodically execute
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -48,3 +48,4 @@ For a complete list of steps, check out the [developer workflow](https://github.
 
 - [ ] I've built the code `cargo build --all-targets` successfully.
 - [ ] I've run the unit tests `cargo test --workspace` and everything passes.
+- [ ] I've made sure my rust toolchain is current `rustup update`.
```
Comment (prince-chrismc) [inline .github/pull_request_template.md:51]: Does this update the extra tools as well? like rustfmt and clippy?

Comment (fishseabowl) [reply .github/pull_request_template.md:51]: Yes. Ref: https://rust-lang.github.io/rustup/concepts/components.html

Comment (efrisch) [reply .github/pull_request_template.md:51]: Yes! And I've added a better screenshot.

**Review by prince-chrismc — APPROVED** [review]
Amazing thanks!

**Review by fishseabowl — APPROVED** [review]
LGTM 👍

**Review by MitaliBo — APPROVED** [review]

**Pull request merged**